lib/agent0/examples/hyperdrive_bots.py

```diff
@@ -0,0 +1,82 @@
+"""Script to showcase running default implemented bots"""
+from __future__ import annotations
+
+import logging
+from typing import TYPE_CHECKING
+
+from agent0 import initialize_accounts
+from agent0.base.config import AgentConfig, Budget, EnvironmentConfig
+from agent0.hyperdrive.exec import run_bots
+from agent0.hyperdrive.policies import Policies
+from fixedpointmath import FixedPoint
+
+if TYPE_CHECKING:
+    from agent0.hyperdrive.agents import HyperdriveWallet
+    from elfpy.markets.hyperdrive import HyperdriveMarket as HyperdriveMarketState
+    from numpy.random._generator import Generator as NumpyGenerator
+
+DEVELOP = True
+ENV_FILE = "hyperdrive_bots.account.env"
+
+env_config = EnvironmentConfig(
+    delete_previous_logs=False,
+    halt_on_errors=True,
+    log_filename="agent0-bots",
+    log_level=logging.INFO,
+    log_stdout=True,
+    random_seed=1234,
+    username="changeme",
+)
+
+agent_config: list[AgentConfig] = [
+    AgentConfig(
+        policy=Policies.random_agent,
+        number_of_agents=3,
+        slippage_tolerance=FixedPoint(0.0001),
+        base_budget=Budget(
+            mean_wei=int(5_000e18),  # 5k base
+            std_wei=int(1_000e18),  # 1k base
+            min_wei=1,  # 1 WEI base
+            max_wei=int(100_000e18),  # 100k base
+        ),
+        eth_budget=Budget(min_wei=int(1e18), max_wei=int(1e18)),
+        init_kwargs={"trade_chance": FixedPoint(0.8)},
+    ),
+    AgentConfig(
+        policy=Policies.long_louie,
+        number_of_agents=0,
+        base_budget=Budget(
+            mean_wei=int(5_000e18),  # 5k base
+            std_wei=int(1_000e18),  # 1k base
+            min_wei=1,  # 1 WEI base
+            max_wei=int(100_000e18),  # 100k base
+        ),
+        eth_budget=Budget(min_wei=int(1e18), max_wei=int(1e18)),
+        init_kwargs={"trade_chance": FixedPoint(0.8), "risk_threshold": FixedPoint(0.9)},
+    ),
+    AgentConfig(
+        policy=Policies.short_sally,
+        number_of_agents=0,
+        base_budget=Budget(
```

Comment:
Can you set one of these to a FixedPoint to illustrate how that works?

Reply:
Can you take a look about the TODO above?

Reply:
this?
```
# TODO should these be in units of wei to match units of Budget?
# Or should budget accept FixedPoints in units of base + eth?
```

Reply:
I don't understand the question. Everything is in wei, I don't know what units of Budget is. Budget generates ints in units of wei

Reply:
that's the one

Reply:
I think everything should be in units of wei